docusaurus/docs/iOS/guides/working-with-attachments.md

```diff
@@ -1,3 +1,263 @@
 ---
 title: Working with Attachments/Files
----
\ No newline at end of file
+---
+
+Stream chat allows you to add attachments to a message. There is built-in support for files, images, videos, giphy and URL preview attachments.
+It is also possible to have your own type of attachment and to customize how built-in attachments are rendered. Custom attachments are one of the most useful ways to add application specific content to messages. Examples of custom attachments are: location sharing, workouts and voice memos.
```

Comment:
> Examples of custom attachments are: location sharing, workouts and voice memos.

Not necessary for an initial release, but when we do a list of common use cases for a feature like this, it can be helpful to treat them as a list of possible scenarios to cover in the guides.

Adding some additional information on voice memos or location sharing may be valuable if we recognize them as main use cases for this feature.